tests/lib.rs

```diff
@@ -22,6 +22,21 @@ fn test_date() {
 
 #[test]
 fn test_millisecond() {
+    let mut i=0;
+    while i < 1000 {
```

Comment:
Can you add a comment that this is a regression test for previous rounding-error prone floating point math?

Reply:
Sure.